.github/workflows/build.yml

```
@@ -2,6 +2,20 @@ name: CI
 on: [push]
 
 jobs:
+    licenses:
+        name: Check licenses headers and dependencies
```

Check licenses headers
```suggestion
        name: Check licenses headers
```